BasicSample/Assets/GrabAndThrow/Materials/White.mat

```diff
@@ -0,0 +1,176 @@
+%YAML 1.1
```

Comment:
Instead of new materials, are we able to reuse any of the MRTK materials? The standard assets package should contain materials for all these colors.

Reply:
yeah you are right, let me change it Kurtis